ddtrace/sampler.py

```diff
@@ -44,8 +45,8 @@
 
 log = get_logger(__name__)
 
-MAX_TRACE_ID = 2 ** 64
-
+# TODO: deprecate and remove MAX_TRACE_ID, it is no longer used.
+MAX_TRACE_ID = _MAX_UINT_64BITS
```

Comment:
this constant is a little bit odd now. Trace IDs can technically be 128 bits now but this values will only be 64 bits. Maybe worth a comment about why it remains 64 bits even though 128 bit support is present?

Reply:
Can we have a comment here explaining this? Thanks!